engine/src/mbldc.cpp

```diff
@@ -241,6 +241,11 @@ void MCScreenDC::handle_motion(MCEventMotionType p_type, double p_timestamp)
 
 void MCScreenDC::process_touch(MCEventTouchPhase p_phase, void *p_touch_handle, int32_t p_timestamp, int32_t p_x, int32_t p_y)
 {
+    if (m_current_window == nil)
```

Comment:
The indentation in the patch is with spaces, whilst the rest of the handler is with tabs.

Reply:
Ah.. Xcode drives me nuts with this. Will fix.

Reply:
Annoyingly the only way I can change to make it tabs was to edit my preferences then re-indent it. You can't even copy/paste a tab from elsewhere. Really odd.